pulp_rpm/app/models/repository.py

```diff
@@ -102,6 +102,71 @@ class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
 
 
+class UlnRemote(Remote):
+    """
+    Remote for "uln" content.
+    """
+
+    TYPE = "uln"
+    uln_server_base_url = models.CharField(
+        max_length=512, default="https://linux-update.oracle.com/"
```

Comment:
This is currently the final item on the list of things I think I could improve about this PR:

I feel a bit uncomfortable defaulting to some Oracle URL right in the DB. I could just not do that, make `uln_server_base_url` a required field in the serializer, and leave everything to the user, but that would not be very user friendly. I think 99.9% of the time, users are going to want to use a single instance wide `uln_server_base_url`, and based on my research (Oracle is the only one I can find in the wild using ULN for content delivery) 99.8% of the time the value will be `https://linux-update.oracle.com/`.

So I have two questions (@goosemania, @bmbouter, or any one else with relevant knowledge):

1. Is there some existing mechanism to let users configure instance wide default values?
2. Is there somewhere else I could default to this value other than directly within the DB?

Reply:
Perhaps I could make it a django setting?
https://docs.pulpproject.org/pulpcore/settings.html#pulp-settings

Reply:
Done.